test/unit/client-daemonset.bats

```diff
@@ -1206,3 +1206,26 @@ rollingUpdate:
   local actual=$(echo $security_context | jq -r .privileged)
   [ "${actual}" = "true" ]
 }
+
+#--------------------------------------------------------------------
+# client.disableHostNodeID
+
+@test "client/DaemonSet: disableHostNodeID defaults to false" {
+  local actual=$(helm template \
```

Comment:
I think it's missing "cd `chart_dir`" like the other tests. (Same for the test below)